### PR TITLE
FIX DOC TST: alpha is an uppper bound on FDR

### PR DESCRIPTION
**Pull request by ogrisel**

### CODE REVIEW

**Comment by amueller** [comment]
That is what I thought. So it should actually be an upper bound.

File: sklearn/feature_selection/tests/test_feature_select.py [hunk 1]
```diff
@@ -402,36 +405,44 @@ def test_select_fdr_regression():
     Test that fdr heuristic actually has low FDR.
     """
     def single_fdr(alpha, n_informative, random_state):
-        n_features = 5 * n_informative
-
-        X, y = make_regression(n_samples=150, n_features=n_features,
+        X, y = make_regression(n_samples=150, n_features=20,
                                n_informative=n_informative, shuffle=False,
-                               random_state=random_state, noise=10)
+                               random_state=random_state, noise=5)
```
Comment (amueller) [inline sklearn/feature_selection/tests/test_feature_select.py]: did you check what this looks like? there is really really little noise...

File: sklearn/feature_selection/tests/test_feature_select.py [hunk 2]
```diff
@@ -402,36 +405,44 @@ def test_select_fdr_regression():
     Test that fdr heuristic actually has low FDR.
     """
     def single_fdr(alpha, n_informative, random_state):
-        n_features = 5 * n_informative
-
-        X, y = make_regression(n_samples=150, n_features=n_features,
+        X, y = make_regression(n_samples=150, n_features=20,
```
Comment (amueller) [inline sklearn/feature_selection/tests/test_feature_select.py]: Was there a particular reason why you fixed the n_features to a constant value?

Comment (ogrisel) [inline sklearn/feature_selection/tests/test_feature_select.py]: It seemed a more interesting thing to test. The relative amount of informative feature does not have to be always the same.

File: sklearn/feature_selection/tests/test_feature_select.py (hunk 1 above)
Comment (ogrisel) [inline sklearn/feature_selection/tests/test_feature_select.py]: I changed that while I was trying to debug. It does not change the outcome of the test.

**Pull request merged**